### ISSUE TITLE
Deathnettles should sting you when you pick them up without gloves.

### ISSUE THREAD

**Issue by Hmeister-real** [space-wizards/space-station-14#25029]
## Description
 Deathnettles should inject 1/4 of what they inject on hit when you pick them up without appropriate gloves, and then fall out of your hand with a message like "It stings your hand!". Should not happen if the person who picks them up is a slime or wearing: any hardsuit (not eva), Insulated gloves, botany gloves, head specific gloves (Caps gloves, Paper-cut proof gloves)

### Additional context
 Deathnettles have 0 downsides other than not piercing Hardsuits, and thats not cool for the most powerful melee weapon in the game by far, since you can get people to crit very very easily as a random civilian. Now only appropriate people will be able to use them.

**Comment by Hmeister-real** [comment]
Credit to tyrfang for the idea, i cant find the original message but its in the deathnettle thread on discord

**Comment by mirrorcult** [comment]
yes, implement, ss13 had it as a balancing factor and it worked well there

**Comment by deltanedas** [comment]
yeah this is funny
rn botany can mass produce deathnettles on warops for every man woman and child, now tiders with no gloves would get trolled

**Comment by UbaserB** [comment]
we need this in our lives

**Comment by Hmeister-real** [comment]
Stale cause of #25068
